app/controllers/api/v1/recipes_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V1::RecipesController < ApplicationController
+  def index
```

Comment:
you may already know this, but we should put this in a service ASAP

Reply:
Okay! I will take a look once I pull this down